### PR TITLE
plumb through notify request handler

### PR DESCRIPTION
**Pull request by maxtaco**
- just forward to the first KBFS client we find

### CODE REVIEW

**Comment by maxtaco** [comment]
r? @strib 

cc: @chrisnojima

File: go/service/notify_fs_request.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
```
Comment (strib) [inline go/service/notify_fs_request.go]: 2016?

**Comment by maxtaco** [comment]
Tons of C/I errors but I don't think they're related

**Comment by maxtaco** [comment]
@strib why didn't these 2 endpoints get the same plumbing as the other 3? Does it need different behavior, or it just wasn't gotten to?

**Comment by strib** [comment]
Which two, the TODOs I mentioned in`kbfs.go`?  Yeah they just weren't gotten to, I only did enough to make it compile when I changed the protocol.

**Comment by maxtaco** [comment]
Do you think it should be the same `kbfsrequest` notify channel, or do you think it should be a different channel?

**Comment by strib** [comment]
Was assuming it would be easiest to make it the same, but no strong opinion.

**Pull request merged**